ui/Screen/Discovery/Project.svelte

```diff
@@ -0,0 +1,116 @@
+<script>
+  import {
+    Avatar,
+    Code,
+    Icon,
+    Text,
+    Title,
+  } from "../../DesignSystem/Primitive";
+  import { Copyable, Stats, TrackToggle } from "../../DesignSystem/Component";
+
+  export let project = null;
+
+  let showTrackButton = false;
+
+  const toggleTrackButton = e => {
+    showTrackButton = e.type === "mouseenter";
+  };
+</script>
+
+<style>
+  .container {
+    border: 1px solid var(--color-foreground-level-2);
+    border-radius: 4px;
+    padding: 32px;
+    overflow: hidden;
+  }
+
+  .container:hover {
+    border: 2px solid var(--color-foreground-level-3);
+    background: var(--color-foreground-level-1);
+  }
```

Comment:
This makes the UI jump cause it's pushing the content around. 
Check out the `Button.svelte` file to see how we solved this with a combination of `box-shadow` and `border`.